src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/dialog/components/content-download-dialog/content-download-dialog.service.ts

```diff
@@ -0,0 +1,25 @@
+import { Injectable } from '@angular/core';
+import { ContentDownloadDialogComponent } from './content-download-dialog.component';
+import { MatDialog, MatDialogRef } from '@angular/material/dialog';
+import { Content } from '@picturepark/sdk-v1-angular';
+
+@Injectable({
```

Comment:
are we going to create a service for every dialog component? it wouldnt be possible to create one generic for all of them where we pass the type of the dialog?